src/course-unit/course-xblock/CourseXblock.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import { Dropdown, Icon, IconButton } from '@edx/paragon';
+import { EditOutline as EditIcon, MoreVert as MoveVertIcon } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from './messages';
+
+const CourseXblock = ({ id, title }) => {
+  const intl = useIntl();
+
+  return (
+    <li className="studio-xblock-wrapper mb-4">
+      <section className="wrapper-xblock p-4">
```

Comment:
[question] maybe `article`?

Reply:
Replaced with the paragon `Card` component